### DISCUSSION TITLE
Communication error during web game start

### DISCUSSION THREAD

**Discussion by danconde** [azadbolour/eagerwords-community#22] [Q&A] [answered]
I think this error should not appear upon initial web app start. I can understand if this occurs when game is in progress and computer goes to sleep and one resumes.
![Screen Shot 2023-05-06 at 7 29 39 PM](https://user-images.githubusercontent.com/4314771/236681971-de58e27e-84c8-4546-a959-675bc87785d9.png)
[eagerwords-test.com-1683426555074.log](https://github.com/azadbolour/eagerwords-community/files/11415144/eagerwords-test.com-1683426555074.log)

**Comment by azadbolour** [comment, marked as answer]
This could still occur at startup if the network is unavailable. But apparently this happened at a time when the network was in fact available. I cannot easily reproduce that scenario. I have added it to my todo list to check the code for the issue and improve logging to provide clues as to why this error is reported when the network is available.
